packages/@ember/-internals/glimmer/lib/resolver.ts

```diff
@@ -99,6 +105,12 @@ function lookupComponentPair(
     }
   }
 
+  if (isRemoved(DEPRECATIONS.DEPRECATE_COMPONENT_TEMPLATE_RESOLVING.options)) {
```

Comment:
You should do `DEPRECATIONS.DEPRECATE_COMPONENT_TEMPLATE_RESOLVING.isRemoved`

Reply:
you got it!